packages/cozy-mespapiers-lib/src/components/ModelSteps/AcquisitionResult.jsx

```diff
@@ -42,7 +42,8 @@ const AcquisitionResult = ({ currentFile, setCurrentFile, currentStep }) => {
 
   const changeSelectedFile = () => {
     const newData = formData.data.filter(
-      data => data.file.name !== currentFile.name
+      data =>
+        data.file.name !== currentFile.name || data.file.id !== currentFile.id
```

Comment:
Shouldn't this also test the `lastModified` date? In case multiple files have the same name?